api.web.yaml

```diff
@@ -542,3 +584,134 @@ components:
         updated_at:
           type: string
           description: Time in RFC3339 format when the asset was last updated
+
+    Study:
+      type: object
+      description: A research study
+      required:
+        - id
+        - owner_user_id
+        - created_at
+        - updated_at
+        - title
+        - description
+        - admin
+        - controller
+        - controller_other
+        - ucl_sponsorship
+        - cag
+        - cag_ref
+        - ethics
+        - hra
+        - iras_id
+        - nhs
+        - nhs_england
+        - nhs_england_ref
+        - mnca
+        - dspt
+        - dbs
+        - data_protection
+        - data_protection_prefix
+        - data_protection_date
+        - data_protection_id
+        - data_protection_number
+        - third_party
+        - external_users
+        - consent
+        - non_consent
+        - ext_eea
```

Comment:
- I have set these to `required` by default as I thought it would be better to have these explicitly set in the database (i.e. even if they are empty strings or `false` by default).

- But we can change this if desired so that optional fields do not get set in the database if they are not explicitly set by the user.